exporters/elasticsearch-exporter/src/test/java/io/camunda/zeebe/exporter/ElasticsearchClientTest.java

```diff
@@ -69,14 +66,7 @@ public void shouldThrowExceptionIfFailToFlushBulk() {
     // when/then
     assertThatThrownBy(client::flush)
         .isInstanceOf(ElasticsearchExporterException.class)
-        .hasMessage("Failed to flush all items of the bulk");
-
-    verify(logSpy)
-        .warn(
-            "Failed to flush {} item(s) of bulk request [type: {}, reason: {}]",
-            bulkSize,
-            "mapper_parsing_exception",
-            "failed to parse");
+        .hasMessageContaining("Failed to flush bulk request:");
```

Comment:
Shouldn't the exception now contain that message?

Reply:
Yeah, though I'm not a fan of asserting the exact message - it's not really part of our API, so why should a test fail if the format slightly changes? But I guess we only have one exception type, so... :man_shrugging:

Reply:
I mean it was before already asserting the log message right ? :sweat_smile: